src/Sylius/Component/Grid/Definition/Grid.php

```diff
@@ -0,0 +1,250 @@
+<?php
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Component\Grid\Definition;
+
+/**
+ * @author Paweł Jędrzejewski <pawel@sylius.org>
+ */
+class Grid
+{
+    /**
+     * @var string
+     */
+    private $code;
+
+    /**
+     * @var string
+     */
+    private $driver;
+
+    /**
+     * @var array
+     */
+    private $driverConfiguration;
+
+    /**
+     * @var array
+     */
+    private $sorting = [];
+
+    /**
+     * @var array
+     */
+    private $fields = array();
+
+    /**
+     * @var array
+     */
+    private $filters = array();
+
+    /**
+     * @var array
+     */
+    private $actionGroups = array();
```

Comment:
Consistency with short-array syntax policy - just pointing this out to not be missed ;)